.github/workflows/update-tags.yml

```diff
@@ -0,0 +1,16 @@
+name: Update tags
+on:
+  release:
+    types: [released]

```

Comment:
Oh, actually, I'd rather this get triggered when a tag is pushed with the pattern `v*.*.*`, instead of when a release is created.

Reply:
Actually I think "release" makes more sense because, remember, the GitHub Action only gets published to the GitHub Actions Marketplace _when a new release is created_. So if you set it to tags, and you just push a tag, but **without a release**, then you've "published" a tag, but it didn't get tracked by GitHub Marketplace.

![image](https://user-images.githubusercontent.com/61068799/225100888-071bc571-1e05-4615-af14-d4917032b61f.png)

Reply:
OK, I'm sold. Just checked and it looks like GH uses this trigger, too.